compiler/tools/fpp-to-cpp/test/component/base/ActiveAsyncProductPortsOnlyComponentAc.ref.cpp

```diff
@@ -367,54 +385,6 @@ void ActiveAsyncProductPortsOnlyComponentBase ::
   // Default: no-op
 }
 
-// ----------------------------------------------------------------------
-// Invocation functions for special output ports
-// ----------------------------------------------------------------------
-
-void ActiveAsyncProductPortsOnlyComponentBase ::
```

Comment:
Where did these go?

Reply:
That component has data product ports but no data products. In the new code gen, the data product invoke functions are private, and they are present only if there are data products.